### DISCUSSION TITLE
Apollo Save Tool v1.2.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps4#60] [Announcements]
### Added

* Local Web server (Download saves as .Zip)
* RAR, Zip, 7-Zip archive extraction (place files on `/data/`)
* 50+ Save Wizard cheat-code files (collected by @OfficialAhmed)
* Custom decryption support
  * Borderlands 3
  * Grand Theft Auto 5 (AES)
  * Resident Evil Revelations 2 (Blowfish)
* Custom checksum support
  * Alien: Isolation
  * Resident Evil Revelations 2 (SHA1)

### Fixed

* Fixed issue when cheat patch couldn't be applied

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps4/releases/tag/v1.2.0'>Apollo Save Tool v1.2.0</a>.</em>

**Comment by Monster023** [comment]
Hi @bucanero , 

i tried loading up my saved data in my HDD but it just keep closing back to the PS4 main page. i dunoe why. please help. 

Thanks
J

**Reply by Monster023** [reply]
Hi @bucanero ,

i am trying to save a new fifa 22 saved files, but i can"T seems to be able to find it. Do you know why the apollo app keeps reading an old saved file?

**Comment by latrinlatrin** [comment]
Hi @bucanero 

i tried to use the RAR extractor on a game but it doesn't seem to work, plus an empty message pops up. Solutions?

**Reply by bucanero** [reply]
I'd suggest to try with a small [test RAR file](https://github.com/bucanero/psptools/releases/download/20220719/SGDeemer111.rar) first. For example: download that link, and place the `SGDeemer.rar` file on your PS4 /data/ folder. It should end up like:
- `/data/sgdeemer.rar`

Next open Apollo, go to the User tools and the `sgdeemer.rar` file should show up there. Try to extract it, and see if it works. If that works, then it might be related to the file you're trying to extract. (this wasn't tested with huge files or multiple part files, etc)

**Comment by depesruts** [comment]
Hi @bucanero

What's the name of the background song that plays while using the program?
Thanks!

**Reply by bucanero** [reply]
> Hi @bucanero
> 
> What's the name of the background song that plays while using the program? Thanks!

It's a theme song from the Kingdom Hearts game. You can find the track here: https://github.com/bucanero/apollo-ps4/blob/main/assets/audio/background_music.mp3

**Reply by Masamune3210** [reply]
Specifically, its Dearly Beloved from Kingdom Hearts 2

**Comment by ddcc5305** [comment]
Can i use this version on ps4 6.72?

**Reply by bucanero** [reply]
> Can i use this version on ps4 6.72?

yes, 6.72 is supported. (GoldHEN 1.1+ or higher required)

**Comment by Peterprogy** [comment]
Hello,
Thank you for your work thank you for your great work which is very practical and appreciable.
Apparently apollo doesn't seem to work with guerrilla games (Horizon 1 &2).
Saves install without problems but the game does not see them
Do you know why and is there a solution?
Thanks

**Reply by bucanero** [reply]
Apollo works fine with Horizon games, people have used it even to transfer saves between the ps4 version and the PC version of the game.

Your issue is probably related to:
- the save TITLE-ID does not match the TITLE-ID of the game
- the save-game keystone doesn't match the keystone of the game
- the save-game is for  a different edition of the game (regular game vs. Complete edition)

**Reply by Peterprogy** [reply]
Good evening,
Thank you for your reply.
Personally I even tried to transfer saves from different accounts but on the same console and it didn't work for Horizon.
I transferred all my backups together and the those from horizon did not work
Backups are copied but the game does not see them...
I do not understand why
is there something to try?
Thanks

**Comment by shangis12** [comment]
Hi @bucanero 
How do i decrypt gta v save game with apollo?
I have tried with few youtube videos and didn't success. 
I want to use the decrypt save file on to gta v save editor without save wizard. 
Thx

**Reply by bucanero** [reply]
if the title-id matches, you should see two options when you open a GTA5 save:
- AES Decrypt memory.dat
- AES Encrypt memory.dat

select only the "decrypt" option, and then "apply changes". You'll have a decrypted data file. Then you can "export" the data file if you want. 
Note that you also need to "encrypt" the data file back before you load that save or the game will fail.

you can also try this decrypter on your PC: https://github.com/bucanero/ps3-save-decrypters/tree/master/gta5-decrypter

**Reply by shangis12** [reply]
When i select only decrypt and apply changes it says error cheat codes
couldn't be applied
I tried on the official save game and on the copied save game on the usb

**Reply by shangis12** [reply]
I have ps4 not ps3

**Reply by bucanero** [reply]
are you sure your save has a `memory.dat` file?

also, you should go to `Settings` and "enable debug logs". Then try to decrypt and check the log file for additional info.

**Reply by shangis12** [reply]
How i make sure that i have memory.dat file?
I use apollo 1.3.1 
And under cheat section i have encrypt and decrypt

**Comment by shangis12** [comment]
[apollo.log](https://github.com/bucanero/apollo-ps4/files/11220332/apollo.log)

Here's my log
Can you tell me how to get memory.dat and what to do?
